code/game/objects/items/kitchen.dm

```diff
@@ -30,7 +30,7 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30)
 	var/datum/reagent/forkload //used to eat omelette
-	var/loaded_egg /// True if the loaded item is an egg, false if its pie
+	var/loaded_food = "nothing" /// The name of the thing on the fork
```

Comment:
Set it to null not "nothing"

Reply:
No, then if something goes wrong its says its a fork full of nothing